src/docs/user/references/ApplicationConnectors.adoc

```diff
@@ -78,3 +78,772 @@ The Informatica workflow consists of a list of variables described in the follow
 This WorkFlow performs a GET request to an Informatica API associated to an application already created by the Informatica user.
 This workflow template supports the following authentication methods: a basic authentication using a username and a password, an advanced authentication using a JWT access token and finally an anonymous authentication where no authorization is required.
 The authentication method is determined based on the definition of the parameters when launching the WF.
+
+
+[#_servicenow]
+=== ServiceNow
+
+The ServiceNow connectors allow ProActive to communicate with ServiceNow instances to perform CRUD operations on their data as well as to send emails.
+
+The installation of the ProActive application in ServiceNow, which enables a ServiceNow instance to communication with a ProActive server, is described
+in the link:../admin/ProActiveAdminGuide.adoc#_servicenow_integration[ServiceNow admin documentation].
+
+==== ProActive to ServiceNow
+
+A set of ProActive workflows are available in the catalog to communicate with ServiceNow.
```

Comment:
A set of ProActive workflow templates

Reply:
Regarding the usage of these connectors, may be mention that they are generally integrated as an activity task in a end-to-end workflow